### ISSUE TITLE
Learning and Generalization in Overparametrized NN, Going Beyond Two Layers

### ISSUE THREAD

**Issue by vshalaeva** [vshalaeva/DL_reading_group#5]
http://papers.nips.cc/paper/8847-learning-and-generalization-in-overparameterized-neural-networks-going-beyond-two-layers.pdf

Abstract
The fundamental learning theory behind neural networks remains largely open. What classes of functions can neural networks actually learn? Why doesn’t the trained network overfit when it is overparameterized?
In this work, we prove that overparameterized neural networks can learn some notable concept classes, including two and three-layer networks with fewer pa- rameters and smooth activations. Moreover, the learning can be simply done by SGD (stochastic gradient descent) or its variants in polynomial time using poly- nomially many samples. The sample complexity can also be almost independent of the number of parameters in the network.
On the technique side, our analysis goes beyond the so-called NTK (neural tan- gent kernel) linearization of neural networks in prior works. We establish a new notion of quadratic approximation of the neural network, and connect it to the SGD theory of escaping saddle points.
